src/libs/antares/study/area/links.cpp

```diff
@@ -416,22 +417,21 @@ bool AreaLinksInternalLoadFromProperty(AreaLink& link, const String& key, const
     return false;
 }
 
-void logLinkDataCheckError(bool& gotFatalError, const AreaLink& link, const String& msg, int hour)
+void logLinkDataCheckError(const AreaLink& link, const String& msg, int hour)
 {
     logs.error() << "Link (" << link.from->name << "/" << link.with->name << "): Invalid values ("
                  << msg << ") for hour " << hour;
-    gotFatalError = true;
+    throw Antares::Error::ReadingStudy();
```

Comment:
The minor drawback of this approach is that we will not list anymore all errors, only the first one.

However, it seems reasonable to me!

Reply:
It is not acceptable from a user's point of view, since they may want to fix as many errors as possible before launching a new simulation, rather than fixing errors one by one.

Reply:
We need to group all these checks in the same part of the code probably